modules/ROOT/pages/_partials/link-anypoint-platform-accounts.adoc

```diff
@@ -0,0 +1,110 @@
+// tag::profileLinkingContextForUsers[]
+[[profile-linking-context-users]]
+= Linking Your Anypoint Platform Profiles
+
+When you log in to Anypoint Platform for the first time using an external identity provider (IdP), Anypoint Platform locates potential user accounts matched by your email address that is associated with the IdP profile. You are prompted to link your IdP profile with the existing accounts. Your options for linking profiles vary based on whether the profiles use the Anypoint identity provider (IdP) or an external IdP that uses single sign-on (SSO). Anypoint Platform supports:
+
+* Linking an SSO login to a local profile by using credentials (login managed by Anypoint IdP)
+* Linking an SSO login to another SSO login that uses a different IdP to log in
+* Linking an SSO login to multiple other profiles that each use different IdPs to log in
+
+Anypoint Platform does not support linking multiple profiles that use the same IdP. The IdP must be an OpenID Connect (OIDC) provider. You can link up to 25 user profiles. This feature is available only if an organization administrator has enabled it.
+
+// end::profileLinkingContextForUsers[]
+
+// tag::profileLinkingForSSOtoLocal[]
+[[sso-to-local]]
+= Linking an SSO Login to a Local Profile
+
+When you attempt to log in using SSO and there is another user profile that shares the same email address but uses Anypoint Platform credentials to log in, you are prompted to link the profiles.
+
+. In the Anypoint Platform login page, click *Sign in with <your identity provider>*.
+. If prompted, enter your credentials for your IdP. +
+If your SSO credentials share an email address with a profile that is managed by the Anypoint IdP, you are prompted to log in. If you are not prompted, your local profile uses a different email address than your SSO login.
```

Comment:
I think we can skip "If you are not prompted, your local profile uses a different email address than your SSO login." Using "your local profile" can be confusing because at the time of login, we/the service doesn't really know whether the customer/end user has an account or not. So let's not imply they do.

Reply:
```suggestion
If your SSO credentials share an email address with a profile that is managed by the Anypoint IdP, you are prompted to log in.
```